graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsDataLoaderRegistry.kt

```diff
@@ -165,34 +165,23 @@ open class DgsDataLoaderRegistry : DataLoaderRegistry() {
      * @return total number of entries that were dispatched from registered [org.dataloader.DataLoader]s.
      */
     override fun dispatchAllWithCount(): Int {
-        var sum = 0
-        scheduledDataLoaderRegistries.forEach {
-            sum += it.value.dispatchAllWithCount()
-        }
-        sum += dataLoaderRegistry.dispatchAllWithCount()
-        return sum
+        return scheduledDataLoaderRegistries.values.sumOf { registry -> registry.dispatchAllWithCount() } +
+            dataLoaders.values.sumOf { dataLoader -> dataLoader.dispatchWithCounts().keysCount }
```

Comment:
Can't we use `super.dispatchAllWithCount` and add that to the sum?

Reply:
I think it would double count, since it calls `getDataLoaders`, which we've overridden to include the dataloaders from `scheduledDataLoaderRegistries`.

Reply:
Oh I see..right. Thanks for that.